app/models/miq_policy.rb

```diff
@@ -11,6 +11,9 @@ class MiqPolicy < ApplicationRecord
                                  PhysicalServer
                                  Vm).freeze
 
+  CONDITION_ALLOW = N_("Success")
+  CONDITION_DENY  = N_("Failure")
```

Comment:
it's weird to me that the constant name is one thing but the text is something else.  maybe:

```suggestion
  CONDITION_SUCCESS = N_("Success")
  CONDITION_FAILURE = N_("Failure")
```

Reply:
Yeah, I'm mixing policy terminology that prefers allow/deny over success/failure but I agree, it doesn't make sense at all here.